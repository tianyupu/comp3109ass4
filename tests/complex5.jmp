  x = 1;
  x = 2;
  x = 3;
  if x goto L2;
  return x;
  y = 5;
L1:
  a = b;
  b = c;
  c = d;
  return x;
L2:
  goto L1;
