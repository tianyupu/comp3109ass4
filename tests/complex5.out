  x = 1;
  x = 2;
  x = 3;
  if x goto L2;
  return x;
L2:
  return x;
